collector/configuration/liverig.properties.md

```diff
@@ -134,4 +141,13 @@ wildcard_period_in_seconds=600
 # crystalfontz_port=/dev/ttyLCD
 # crystalfontz_baud=115200
 
+# WITSML Change mechanism features
+#
+# witsml_change_mechanism_feature=true
+# witsml_change_mechanism_feature.witsml_change_mechanism_delay_period_in_seconds=2
+# witsml_change_mechanism_feature.witsml_change_mechanism_detection_period_in_seconds=10
+# witsml_change_mechanism_feature.witsml_change_mechanism_initial_last_change_feature=false
+# witsml_change_mechanism_feature.witsml_change_mechanism_late_channels_feature=true
+# witsml_change_mechanism_feature.witsml_change_mechanism_object_growing_feature=true
+# witsml_change_mechanism_feature.witsml_change_mechanism_initial_last_change_in_minutes=10
```

Comment:
Please fix here also.. probably you copied from "test plan" routine but not from the original source code.

Here it's not a documentation how to enable the new feature but the documentation about the default values and property names

```java
    public static final boolean DEFAULT_WITSML_CHANGE_MECHANISM_FEATURE = false;
    public static final boolean DEFAULT_WITSML_CHANGE_MECHANISM_LATE_CHANNELS_FEATURE = false;
    public static final boolean DEFAULT_WITSML_CHANGE_MECHANISM_OBJECT_GROWING_FEATURE = false;
    public static final long DEFAULT_WITSML_CHANGE_MECHANISM_DETECTION_PERIOD_IN_SECONDS = 10;
    public static final boolean DEFAULT_WITSML_CHANGE_MECHANISM_INITIAL_LAST_CHANGE_FEATURE = true;
    public static final long DEFAULT_WITSML_CHANGE_MECHANISM_INITIAL_LAST_CHANGE_IN_MINUTES = 1440;
    public static final long DEFAULT_WITSML_CHANGE_DELAY_PERIOD_IN_SECONDS = 0;
```